components/brave_wallet_ui/components/extension/connect-hardware-wallet-panel/index.tsx

```diff
@@ -169,6 +171,10 @@ export const ConnectHardwareWalletPanel = ({
   // custom hooks
   useInterval(retryHardwareOperation, 3000, !isConnected ? 5000 : null)
 
+  React.useEffect(() => {
+    history.push('')
+  }, [history])
```

Comment:
Let's add a comment as to why this is needed so that it is not removed.

Also, will this work without `history` in the deps array?

Reply:
Added comment and removed `history` from array